docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -553,7 +554,9 @@ The impersonation rules are a bit different than the other rules: The attribute
 Doing so it was possible to make the attribute ``allow`` optional.
 
 The following example allows the ``admin`` role, to impersonate any user, except
-for ``bob``. It also allows any user to impersonate the ``test`` user:
+for ``bob``. It also allows any user to impersonate the ``test`` user. It also
```

Comment:
Provide first generic explanation of substitution and then use example with `backend`

Reply:
Done.

I have added more detailed generic explanations on lines 547-549 above, as that section is more suited for generic explanations. I have left this line untouched because this section specifically focuses on explaining the examples included below (`user-impersonation.json`).